src/main/java/com/renatusnetwork/momentum/data/stats/StatsManager.java

```diff
@@ -862,10 +863,14 @@ public void resetPracticeDataOnly(PlayerStats playerStats)
         if (playerStats.inPracticeMode())
         {
             Player player = playerStats.getPlayer();
+            PlayerInventory inv = player.getInventory();
             ItemStack item = Utils.getPracPlateIfExists(player.getInventory());
 
-            if (item != null)
-                player.getInventory().remove(item);
+            if (item != null) {
```

Comment:
while this is good, we are also going to want to do this offhand checking for leave item removal if it doesn't already exist. I believe sword/shield does this so no need to add it there but make sure of that too

Reply:
leaving a level with redstone dust in offhand gets handled now
leaving a level with another item in offhand (prismarine for example) will keep it in offhand